Interop/Runtime/src/jvm/kotlin/kotlinx/cinterop/JvmCallbacks.kt

```diff
@@ -453,6 +495,8 @@ private fun ffiCreateClosure(ffiCif: ffi_cif, impl: FfiClosureImpl): NativePtr {
         -1L -> throw Error("libffi error occurred")
     }
 
+    caches.addClosure(res)
```

Comment:
`userDataGlobalRef` in `ffiCreateClosure0` seems to leak.
Closure itself seems to leak if `ffi_prep_closure_loc` fails.